site/using-cluster-operator.md

```diff
@@ -0,0 +1,790 @@
+`Role` and `RoleBinding`, it will only be effective in the Namespace where the RBACs are deployed.
+
+If you do not have Pod security policies enabled, skip to <a href="#create">Create a RabbitMQ Instance</a> below.
+
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#example) has an example
+to create RBAC rules and a policy.
+
+## <a id='create' class='anchor' href='#create'>Create a RabbitMQ Instance</a>
+
+To create a RabbitMQ instance, you must create a `RabbitmqCluster` resource definition and apply it.
+Cluster Operator creates the necessary resources, such as Services and StatefulSet, in the same namespace
+in which the `RabbitmqCluster` was defined.
+
+1. Create a YAML file to define a `RabbitmqCluster` resource named `definition.yaml`.
+
+    <p class="note">
+      <strong>Note:</strong> The YAML file can have any name, but the steps that follow assume it is named
+      <code>definition</code>.
+    </p>
+
+1. Copy and paste the below snippet into the file and save it.
+
+    <pre class='hljs'>
+    apiVersion: rabbitmq.com/v1beta1
+    kind: RabbitmqCluster
+    metadata:
+      name: definition
+    </pre>
+
+1. Apply the definition by running:
+
+    ```
+    kubectl apply -f definition.yaml
+    ```
+
+1. Verify that the process was successful by running:
+
+    ```
+    kubectl get all -l app.kubernetes.io/name=definition
+    ```
+
+    If successful, you see a running Pod and a service that exposes the instance.
+
+    For example:
+
+    <pre class="terminal">
+    $ kubectl get all -l app.kubernetes.io/name=definition
+    NAME                               READY   STATUS    RESTARTS   AGE
+    pod/definition-rabbitmq-server-0   1/1     Running   0          112s
+    </pre>
+
+    <pre class="terminal">
+    NAME                                   TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                        AGE
+    service/definition-rabbitmq-headless   ClusterIP   None             None        4369/TCP                       113s
+    service/definition-rabbitmq-ingress    ClusterIP   10.103.214.196   None        5672/TCP,15672/TCP,15692/TCP   113s
+    </pre>
+
+You can now start using RabbitMQ in your app or continue with more advanced configuration options.
+For more information, see the [RabbitMQ documentation](https://www.rabbitmq.com/getstarted.html).
+
+## <a id='configure' class='anchor' href='#configure'>Configure a RabbitMQ Instance</a>
+
+To configure a RabbitMQ instance:
+
+1. Open `definition.yaml` or edit the the configuration in place by running:
+
+    ```
+    kubectl edit rabbitmqcluster definition
+    ```
+
+1. Add any of the properties described below along with their values. Every property listed below is optional.
+
+### <a name='replicas' class='anchor' href='#replicas'>Replicas</a>
+
+**Description:** Specify the number of replicas for the RabbitmqCluster.
+
+**Default Value:** 1
```

Comment:
We must mention that [two node clusters are not supported](https://www.rabbitmq.com/clustering.html#node-count) by a lot of features and are highly recommended against.